buildscripts/azure/azure-windows.yml

```diff
@@ -0,0 +1,54 @@
+parameters:
+  name: ''
+  vmImage: ''
+
+jobs:
+- job: ${{ parameters.name }}
+  pool: 
+    vmImage: ${{ parameters.vmImage }}
+  strategy:
+    maxParallel: 11
+    matrix:
+      py27:
+        PYTHON: '2.7'
+        CONDA_ENV: travisci
+      py37:
+        PYTHON: '3.7'
+        CONDA_ENV: travisci
+
+  steps:
+
+    - powershell: |
+        $wc = New-Object net.webclient
+        $wc.Downloadfile("https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe", "Miniconda3-latest-Windows-x86_64.exe")
+        Start-Process "Miniconda3-latest-Windows-x86_64.exe" "/S /D=C:\Miniconda3" -Wait
+      displayName: 'Install miniconda'
+
+    # Need to install VC 9.0 only for Python 2.7
+    - powershell: |
+        $wc = New-Object net.webclient
+        $wc.Downloadfile("https://download.microsoft.com/download/7/9/6/796EF2E4-801B-4FC4-AB28-B59FBF6D907B/VCForPython27.msi", "VCForPython27.msi")
+        Start-Process "VCForPython27.msi" /qn -Wait
+      displayName: 'Install VC 9.0'
+      condition: eq(variables['PYTHON'], '2.7')
+
+    - script: |
+        call C:\Miniconda3\Scripts\activate.bat
+        call activate base
+        call conda list
+        call conda remove --all -q -y -n %CONDA_ENV%
+        call conda create -n %CONDA_ENV% -q -y python=%PYTHON% cmake
+        call activate %CONDA_ENV%
+        call conda install -y -q -c numba llvmdev="6.0*"
+        if %PYTHON% LSS 3.4 (call conda install -y -q enum34)
+      displayName: 'Before Install'
```

Comment:
Out of interest... why isn't this calling the script with this contents? `buildscripts/incremental/setup_conda_environment.sh` ?

Reply:
I assume you mean `buildscripts/incremental/setup_conda_environment.cmd` (since this is Windows)?   The answer is that script has several calls to `conda` which are not prefixed by `call`.  For reasons I don't fully understand, that causes the `setup_conda_environment.cmd` to exit after the first success, and stop running the rest of the commands.  After several baffling rounds, I finally gave up and pull the commands into the Azure Pipeline YAML file directly so I could prefix them all with `call`.

I have no idea why the setup file doesn't work on Azure and simultaneously don't understand why it does work on AppVeyor, but it doesn't matter at this point.  :)

Reply:
ah yes, `.sh` is muscle memory :) This is indeed strange. So long as we only have to maintain one, that's fine!